readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -209,29 +182,24 @@ function attach_elastic_search_query_sphinx(data) {
 
                             // Create some spacing between the results.
                             // Also, don't add this spacing in the last hit.
-                            if (j !== inner_hits.length - 1) {
+                            if (j < blocks.length - 1) {
                                 list_item.append($("<div class='rtd_search_hits_spacing'></div>"));
                             }
                         }
 
                         Search.output.append(list_item);
                         list_item.slideDown(5);
                     }
-                }
-
-                if (!hit_list.length) {
-                    // Fallback to Sphinx's indexes
-                    Search.query_fallback(query);
-                    console.log('Read the Docs search failed. Falling back to Sphinx search.');
-                }
-                else {
                     Search.status.text(
-                        _('Search finished, found %s page(s) matching the search query.').replace('%s', hit_list.length)
+                        _('Search finished, found %s page(s) matching the search query.').replace('%s', results.length)
                     );
+                } else {
+                    console.log('Read the Docs search failed. Falling back to Sphinx search.');
+                    Search.query_fallback(query);
                 }
             })
             .fail(function (error) {
-                // Fallback to Sphinx's indexes
+                console.log('Read the Docs search failed. Falling back to Sphinx search.');
```

Comment:
👍  Might be good to do this at a debug level. 
```suggestion
                console.debug('Read the Docs search failed. Falling back to Sphinx search.');
```